enterprise_access/apps/bffs/checkout/handlers.py

```diff
@@ -38,6 +37,11 @@
 logger = logging.getLogger(__name__)
 
 
+def get_stripe_checkout_session(session_id):
```

Comment:
there's already an identical function in `customer_billing/stripe_api.py`

Reply:
The problem is that function is wrapped in a cacher, the first call gets cached as null. I did add a skip_cache option, but it felt a bit cluttered. Open to other suggestions.